templates/firefox-ios.xliff

```diff
@@ -2713,7 +2536,62 @@
         <source>$(PRODUCT_NAME)</source>
       </trans-unit>
       <trans-unit id="CFBundleShortVersionString">
-        <source>14.0</source>
+        <source>15.0</source>
+      </trans-unit>
+    </body>
+  </file>
+  <file original="Shared/Supporting Files/Intro.strings" source-language="en" datatype="plaintext">
+    <header>
+      <tool tool-id="com.apple.dt.xcode" tool-name="Xcode" tool-version="8.3.3" build-num="8E3004b"/>
+    </header>
+    <body>
+      <trans-unit id="Intro.Slides.Mail.Description">
+        <source>Use any email app — not just Mail — with Firefox.</source>
+        <note>Description for the 'Mail' panel in the First Run tour.</note>
+      </trans-unit>
+      <trans-unit id="Intro.Slides.Mail.Title">
+        <source>You’ve got mail… options</source>
```

Comment:
I'm wondering what the "options" here is referring to? Probably good to call out

Reply:
This was just the way the copy was written? Not sure how I can make this more clear 😢